modules/utxo-ord/src/inscriptions.ts

```diff
@@ -57,8 +66,72 @@ export function createOutputScriptForInscription(
     output: compiledScript,
     depth: 0,
   };
-  const payment = payments.p2tr({ redeems: [redeem] }, { eccLib });
+
+  // cal fees here
+  return payments.p2tr({ redeems: [redeem] }, { eccLib });
+}
+
+export type InscriptionRevealData = {
+  address: string;
+  revealSize: number;
+};
+
+/**
+ * @returns InscriptionRevealData
+ * @param createInscriptionParams & network
+ */
+export function createInscriptionRevealData(
+  createInscriptionParams: CreateInscriptionParams & { network: Network }
+): InscriptionRevealData {
+  const payment = createPaymentForInscription(createInscriptionParams);
+
+  const commitRecipient = payments.p2tr(payment, { eccLib });
+  const { output: commitOutput } = commitRecipient;
```

Comment:
```suggestion
  const commitRecipient = payments.p2tr(payment, { eccLib });
  const { output: commitOutput } = commitRecipient;
```
const { output: commitOutput } = payment;